src/features/blog/containers/newsContainer/index.js

```diff
@@ -10,75 +10,68 @@ import {
   Row,
   GreyWrapper,
 } from '../../../../components/grid'
-// import { ShowMoreButton } from './styles'
 import Button from '../../../../components/button'
 
 const initialPaginationState = {
   end: 9,
   step: 9,
 }
-class NewsContainer extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      selectedFilter: props.categories.find(
-        ({ title }) => title === 'All Articles'
-      ),
-      ...initialPaginationState,
-      total: props.articles,
-    }
-  }
+const NewsContainer = ({ articles, categories }) => {
+  const [selectedFilter, setSelectedFilter] = useState(
+    categories.find(({ title }) => title === 'All Articles')
+  )
+  const [initialPagination, setInitialPagination] = useState(
```

Comment:
Could you rename this to reflect better what this function is doing? It's updating the general state of pagination, not just the initial state of pagination.

Reply:
Changed to pagination